### ISSUE TITLE
a11y of expandable/collapsible pattern associated with URI Dereferencer

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/UriDereferencer#18]
As detailed in [omeka/omeka-s#2287](https://github.com/omeka/omeka-s/issues/2287), an accessible pattern should be implemented for instances of expandable/collapsible content. 

Concerning this issue, there are some URI Deferencer-specific issues and suggestions surrounding the existing expand/collapse functionality:

- Expanding and collapsing trigger links is generally moving focus to the web content area.
  - However, if content is loaded into a description list (as opposed to fetching returning an “undefined” label), collapsing does not immediately move focus to the web content area. Instead, a jump menu is prompted that allows one to jump to a group, which then moves focus to the web content area.
- Activating the collapse/expand trigger link (that had actually fetched content) reads out the item ID, which should not be happening.
- Accessible names are missing for trigger links, and accessible descriptions may also be warranted here. Link purposes must be explicitly stated.
- The loader for URI content should announce loading status to screen readers (likely via `aria-label` and `role=“alert”`) when focus is moved to content after expanding (especially important in cases of longer loading periods). It is hard to test this since focus does not easily move through the current setup, but looking at the code, I am not seeing any alerts.
- In parsing, `div` containers that appear when trigger links are expanded are being flagged for improper use on `span` elements in this particular context of use.
